assets/src/console/pages/payments/RefundPage.tsx

```diff
@@ -64,6 +65,22 @@ const RefundPage: React.FC<Props> = ({
 
   const paymentAmount = `${(amount / 100).toFixed(2)}`;
 
+  const paymentRefundQuery = useGetPaymentRefundQuery({
```

Comment:
best to group hooks together before logic starts, so i'd move this up to just below `const [submitRefund, submitRefundResult] = useSubmitRefundMutation();`

Reply:
Addressed all review comments